.structignore

```diff
@@ -5,5 +5,5 @@ test/projects/assorted/@sls-api/test/errors.spec.js
 test/projects/assorted/@sls-api/test/routes.spec.js
 test/projects/assorted/@sls-api/test/handler.spec.js
 test/projects/assorted/@sls-queue/test/queue/queue.spec.js
-test/projects/assorted/@sls-closedsource/test/sls-cf-stack.spec.js
-test/projects/assorted/@sls-closedsource-react/test/sls-cf-stack.spec.js
+test/projects/assorted/@sls-closedsource/test/0_sls-cf-stack.spec.js
+test/projects/assorted/@sls-closedsource-react/test/0_sls-cf-stack.spec.js
```

Comment:
Renamed this too preventing in frontend they would have the same problem...

Reply:
There is a frontend? Seriously though we have no idea what you are talking about :)

Reply:
As I saw `react` here I supposed that this plugin was used by `frontend` team too, that's why the note...